docs/dev_database/common/cell_data.rst

```diff
@@ -0,0 +1,67 @@
+===============
+cell data files
+===============
+
+The *cell data* files are meant for specific primitives which have a common attribute format. The data contained in these files is generated/copied from the corresponding primitives' fuzzers.
+
+Naming convention
+-----------------
+
+The naming scheme for the cell data files is the following::
+
+    <primitive_name>_<data_file_type>.json
+
+Example files:
+
+    - ``gtpe2_common_attrs.json``
+    - ``gtpe2_channel_ports.json``
+
+File format
+-----------
+
+There are two main data file types:
+
+    - *Ports*
+    - *Attributes*
+
+Attributes files
+~~~~~~~~~~~~~~~~
+
+This is a JSON file containing a dictionary of parameters, each one with, at most, four attributes:
+
+    - Type: one of BIN, INT, STR, BOOL.
+    - Values: all possible values that this parameter can assume. In case of `BIN` types, the values list contains only the maximum value reachable.
+    - Digits: number of digits (or bits) required to enable a parameter.
+    - Encoding: This is present only for `INT` types of parameters. These reflect the actual encoding of the parameter value in the bit array.
```

Comment:
I think a couple examples are going to help here.

Reply:
Done